provisioning/roles/mysql/tasks/main.yml

```diff
@@ -34,6 +34,10 @@
   when: ansible_os_family == "Debian"
   tags: [mysql,packages]
 
+- name: Tune for eZ development
+  copy: src=ez.cnf dest=/etc/mysql/conf.d/ez.cnf owner=root group=root mode=0644
+  notify: restart mysql
+
```

Comment:
Move this segment after the next one.
The flow should go
"Ensure mysql service is up"
"Tune for ez development"

Please add a switch
when: use_mysql_dev == true

Set the default value in https://github.com/dkorunic/vagrant-ansible-ezpublish/blob/master/provisioning/roles/mysql/defaults/main.yml 

and change https://github.com/dkorunic/vagrant-ansible-ezpublish/blob/master/provisioning/vagrant.yml#L11
to - { role: mysql, use_mysql_dev: true}